### ISSUE TITLE
Review for HW2

### ISSUE THREAD

**Issue by ridler** [antsankov/data-engineering-HW1#2]
# Criticism
- You should publish better documentation for the API.
- The database code for the connection and schema could be more expressive / less cryptic.

# Compliment
- I like how you used bash for testing.  It's kind of creative and shows what's happening with the app better than a testing framework would.

**Comment by antsankov** [comment]
Responded to the second criticism. Does it make a little more sense now? It is working on MongoDB which isn't as strict about defining a schema as SQL. This might be why it was a little confusing.

**Comment by ridler** [comment]
Yes it makes more sense now that I know you're using a cloud service.  At first I thought Mongo was running locally.  Good job!
